app/src/utils/plotting.py

```diff
@@ -245,8 +260,65 @@ def plot_flow_aep(
             ],
         )
     )
+    # Add confidence limits for gage AMS if provided
+    if gage_ams_confidence_df is not None and not gage_ams_confidence_df.empty:
+        for label in ["upper", "lower", "computed"]:
+            if label == "computed":
+                plot_name = label
```

Comment:
Can we make the label `computed` more descriptive in the actual plot? Computed based on what?